azDevOps/azure/templates/v2/steps/build-node.yml

```diff
@@ -54,17 +55,20 @@ steps:
       login_azure: true
 
   # Audit dependencies
-  - ${{ if eq(parameters.build_audit, true) }}:
+  # Important: can only audit public packages in Azure Pipelines
+  - ${{ if ne(parameters.customRegistry, 'useNpmrc') }}:
```

Comment:
people could use registries other than TFS. 
so maybe let's leave the build_audit in there as well as an agnostic override set specifically in the pipeline